tests/solids/diamondC_1x1x1-Gaussian_pp/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+
+
+
+IF(ENABLE_SOA)
+
+  LIST(APPEND DIAMOND_SCALARS "totenergy" "-10.4957 0.0061")
+  LIST(APPEND DIAMOND_SCALARS "kinetic" "11.517 0.055")
+  LIST(APPEND DIAMOND_SCALARS "potential" "-22.013 0.054")
+  LIST(APPEND DIAMOND_SCALARS "eeenergy" "-2.679 0.013")
+  LIST(APPEND DIAMOND_SCALARS "ionion" "-12.78 0.00")
+  LIST(APPEND DIAMOND_SCALARS "localecp" "-7.196 0.060")
+  LIST(APPEND DIAMOND_SCALARS "nonlocalecp" "0.638 0.025")
+  LIST(APPEND DIAMOND_SCALARS "samples" "128000 0.0")
+#  LIST(APPEND DIAMOND_SCALARS "flux" "0.0 0.4")
+
+# Reference VMC run in qmc-ref "-10.496089 0.000159"
```

Comment:
Move the VMC reference value above and update all the values according to your ref run.